### ISSUE TITLE
Bold text on download link

### ISSUE THREAD

**Issue by jungle-boogie** [sqlitebrowser/website#8]
Hi,

This may be by design so I hope this isn't offensive to anyone...

https://sqlitebrowser.org/

The Download text in the header is bold, like it's the selected "tab" on the webpage.

Maybe you want it to be bold, to grab attention and make it faster for people to find how to get the software?

**Comment by justinclift** [comment]
Yeah, made it bold so it kind of stands out.  99% of people coming to the website hit that page, so might as well... :smile:
